ui/src/app/shared/common.ts

```diff
@@ -1,21 +1,28 @@
 /** A collection of enums and static functions. */
 
+/** Enum mapping of statuses and the app-wide icons representing with them. */
 export enum JobStatusImage {
-  Submitted = <any> 'https://www.gstatic.com/images/icons/material/system/1x/file_upload_grey600_24dp.png',
+  Submitted = <any> 'https://www.gstatic.com/images/icons/material/system/1x/remove_grey600_24dp.png',
   Running = <any> 'https://www.gstatic.com/images/icons/material/system/1x/autorenew_grey600_24dp.png',
   Aborting = <any> 'https://www.gstatic.com/images/icons/material/system/1x/report_problem_grey600_24dp.png',
   Failed = <any> 'https://www.gstatic.com/images/icons/material/system/1x/close_grey600_24dp.png',
   Succeeded = <any> 'https://www.gstatic.com/images/icons/material/system/1x/done_grey600_24dp.png',
   Aborted = <any> 'https://www.gstatic.com/images/icons/material/system/1x/report_problem_grey600_24dp.png'
 }
 
-export enum StatusGroup {
-  Active = <any> 'active',
-  Failed = <any> 'failed',
-  Completed = <any> 'completed'
+/** Constant strings for non-configurable primary fields that can be queried over. Each should have a corresponding
```

Comment:
tiny nit: shorten this line so it doesn't wrap since you already have another line of comments.

Reply:
Yup thanks.